platform/ios/ios.xcodeproj/project.pbxproj

```diff
@@ -1008,6 +1008,7 @@
 		96E0272C1E57C7E5004B8E66 /* fr */ = {isa = PBXFileReference; lastKnownFileType = text.plist.strings; name = fr; path = fr.lproj/Localizable.strings; sourceTree = "<group>"; };
 		96E0272D1E57C7E6004B8E66 /* vi */ = {isa = PBXFileReference; lastKnownFileType = text.plist.strings; name = vi; path = vi.lproj/Localizable.strings; sourceTree = "<group>"; };
 		96E0272E1E57C7E7004B8E66 /* pt-BR */ = {isa = PBXFileReference; lastKnownFileType = text.plist.strings; name = "pt-BR"; path = "pt-BR.lproj/Localizable.strings"; sourceTree = "<group>"; };
+		96F017292118FBAE00892778 /* MGLMapView_Experimental.h */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.h; path = MGLMapView_Experimental.h; sourceTree = "<group>"; };
```

Comment:
Xcode apparently doesn’t consider MGLMapView_Experimental.h to be a counterpart of MGLMapView.h. At least in previous versions of Xcode, only certain `_` suffixes were treated as counterparts, such as “Internal” or “Private”. If you want to be more descriptive, you’d have to use a `+` prefix.

/ref #10820

Reply:
I didn’t use `MGLMapView_Private.h` for organizational reasons, but also because that header includes Obj-C++ (so it can’t be used directly in `iosapp`, which so far is pure Obj-C).

Reply:
I’m starting to reconsider the decision to have a separate “experimental” header — we could just have an experimental section in `MGLMapView_Private.h` and make `iosapp` build as Obj-C++ (where appropriate).

I guess it comes down to whether or not we find the separate file to be useful or not.

Reply:
Either works for me. Slightly tempted to keep the experimental file for the tempted as we potentially add other things.

Reply:
Let’s avoid exposing any C++ publicly, per the contributing guidelines. If there needs to be an extra header, so be it.